src/likelihoods/negativebinomial.jl

```diff
@@ -0,0 +1,26 @@
+"""
+    NegBinomialLikelihood(r::Real=1.0, l=logistic)
+
+Negative binomial likelihood with number of successes `r`.
+
+```math
+    p(k|f) = \\operatorname{NB}(k | r, l(f))
```

Comment:
Could you write the explicit formulation of the pdf. The `NegativeBinomial` has multiple formulations and it would good to clarify it.

Reply:
Sure!